packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -29,23 +29,49 @@ const propTypes = {
    */
   title: PropTypes.string,
   /**
-   * Message of the notification-dialog.
+   * Deprecated, Message of the notification-dialog
```

Comment:
nit, can we also mention which new props take precedence over the deprecated ones?

Reply:
Changed it so after the description it says `resolve to ____` https://github.com/cerner/terra-framework/pull/899/commits/cd5d87726ffa459e43be295823a935fea67d1f7a?file-filters%5B%5D=.jsx&file-filters%5B%5D=.md#diff-cb62dc7ce0197bc569a9952450bb9fe5R32